tests/src/test/java/alluxio/client/rest/AlluxioMasterRestApiTest.java

```diff
@@ -191,4 +201,697 @@ public void getWorkers() throws Exception {
+            + ".254`,`right`:`RUNTIME`},{`left`:`alluxio.worker.web.hostname`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`alluxio.worker.web.port`,`middle`:`0`,"
+            + "`right`:`RUNTIME`},{`left`:`alluxio.zookeeper.address`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.connection.timeout`,`middle`:`15s`,"
+            + "`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.election.path`,`middle`:`/election`,"
+            + "`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.enabled`,`middle`:`false`,"
+            + "`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.job.election.path`,"
+            + "`middle`:`/job_election`,`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.job.leader"
+            + ".path`,`middle`:`/job_leader`,`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.leader"
+            + ".inquiry.retry`,`middle`:`10`,`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.leader"
+            + ".path`,`middle`:`/leader`,`right`:`DEFAULT`},{`left`:`alluxio.zookeeper.session"
+            + ".timeout`,`middle`:`60s`,`right`:`DEFAULT`},{`left`:`aws.accessKeyId`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`aws.secretKey`,`middle`:``,`right`:`DEFAULT`},"
+            + "{`left`:`fs.cos.access.key`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.cos.app"
+            + ".id`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.cos.connection.max`,"
+            + "`middle`:`1024`,`right`:`DEFAULT`},{`left`:`fs.cos.connection.timeout`,"
+            + "`middle`:`50sec`,`right`:`DEFAULT`},{`left`:`fs.cos.region`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`fs.cos.secret.key`,`middle`:``,`right`:`DEFAULT`},"
+            + "{`left`:`fs.cos.socket.timeout`,`middle`:`50sec`,`right`:`DEFAULT`},{`left`:`fs"
+            + ".gcs.accessKeyId`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.gcs.secretAccessKey`,"
+            + "`middle`:``,`right`:`DEFAULT`},{`left`:`fs.kodo.accesskey`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`fs.kodo.secretkey`,`middle`:``,`right`:`DEFAULT`},"
+            + "{`left`:`fs.oss.accessKeyId`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.oss"
+            + ".accessKeySecret`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.oss.endpoint`,"
+            + "`middle`:``,`right`:`DEFAULT`},{`left`:`fs.swift.apikey`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`fs.swift.auth.method`,`middle`:``,`right`:`DEFAULT`},"
+            + "{`left`:`fs.swift.auth.url`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.swift"
+            + ".password`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.swift.region`,`middle`:``,"
+            + "`right`:`DEFAULT`},{`left`:`fs.swift.simulation`,`middle`:``,`right`:`DEFAULT`},"
+            + "{`left`:`fs.swift.tenant`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.swift.use"
+            + ".public.url`,`middle`:``,`right`:`DEFAULT`},{`left`:`fs.swift.user`,`middle`:``,"
+            + "`right`:`DEFAULT`}],`whitelist`:[`/`]}").replace('`', '"');
```

Comment:
How do we maintain this list? I imagine this will change as we add more configuration values.